config.json

```diff
@@ -243,6 +243,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 1
+      },
+      {
+        "slug": "change",
+        "name": "Change",
+        "uuid": "7e1807fa-a305-4412-9a53-ed51c484e66c",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
Did you mean to add this entry in this PR?

Reply:
Nope. I shouldn't be doing late PRs I guess :)